src/quo/components/wallet/account_permissions/view.cljs

```diff
@@ -42,17 +42,19 @@
 
 (defn- view-internal
   [{:keys
-    [checked? disabled? on-change token-details keycard? theme container-style]
+    [checked? disabled? on-change token-details keycard? theme container-style customization-color]
     {:keys
-     [name address emoji customization-color]} :account}]
+     [name address emoji]
+     :as account} :account
+    :or {customization-color :blue}}]
   [rn/view
    {:style               (merge (style/container theme) container-style)
     :accessibility-label :wallet-account-permissions}
    [rn/view {:style style/row1}
     [account-avatar/view
      {:size                32
       :emoji               emoji
-      :customization-color customization-color}]
+      :customization-color (:customization-color account)}]
```

Comment:
do you not have access to `customization-color` already? (above seems to be destructured)

Reply:
The `customization-color` extracted here and passed to the `account-avatar` component is specific to each account. Each account can have a different customization color, which is used for the background of the account emoji. On the other hand, the `customization-color` destructured in the function header is the customization color of the `account-permissions` component, which will be used for the checkbox.

<img width="322" alt="Screenshot 2024-02-16 at 07 32 32" src="https://github.com/status-im/status-mobile/assets/19279756/30adc719-956e-4a11-b529-c0f0a6ee0ca2">